float64 vx
float64 vy
float64 vz
float64 fom
float64 altitude
waterlinked_a50/Transducer[] transducers
bool velocity_valid
int32 status
string format
string type
float64 time
float64[9] covariance
uint64 time_of_validity
uint64 time_of_transmission
